### PR TITLE
Calculator layout

### PR DESCRIPTION
**Pull request by FilbertKimasa**
In this pull request, the following was done:

- [ ] Create calculator layout

- [ ] Delete irrelevant code

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      0
+    </button>
+    <button type="button" className="grid-box">
+      .
+    </button>
+    <button type="button" className="grid-box left">
+      =
+    </button>
+  </div>
+);
```
Comment (VICTOR-CHUKWU) [inline src/components/Calculator.js]: - [ ] well done starting this project correctly, according to the project requirements, the Calculator.js component should at least have one child component, this is a way to teach you about building reusable components in React, kindly make those buttons a component and import it here :+1:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @FilbertKimasa  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- all boilerplate files removed
- created a folder called components
- Calculator.js imported into App.js


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (FilbertKimasa) [reply src/components/Calculator.js]: Okay👍

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: :raising_hand: @FilbertKimasa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

:tada: :champagne:  :tada: :champagne: 

## Highlights :green_circle: 

- Calculator component has at least 1 child component as requested :+1: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**